contrib/bloop/test/src/mill/contrib/bloop/BloopTests.scala

```diff
@@ -60,27 +58,26 @@ object BloopTests extends TestSuite {
     }
 
     object scalajsModule extends scalajslib.ScalaJSModule with testBloop.Module {
-      val sv = sys.props.getOrElse("TEST_SCALA_2_13_VERSION", ???)
-      val sjsv = sys.props.getOrElse("TEST_SCALAJS_VERSION", ???)
+      val sv = sys.props("TEST_SCALA_2_13_VERSION")
+      val sjsv = sys.props("TEST_SCALAJS_VERSION")
```

Comment:
This change means, we silently go with a `null`, making any test failure hard to trace back to a missing sysprop. The explicit `???` ensured we'd see the cause, the missing sysprop, early. Sysprops are an easy way to configure the test suite from the build tool, but they can be easily broken, too. Can you revert to the `sys.prop.getOrElse` notation in all places?

Reply:
Huh I didn't realize `sys.props("doesntexist")` returns `null`. I thought it would throw since `sys.props` is a `Map`, but I guess there's a default value. I can revert those changes